doc/manuals/admin/management_api.md

```diff
@@ -1,5 +1,6 @@
 # Management REST interface
 
+## Log and Trace levels
```

Comment:
Note that I have made 'Log and Trace levels' (well, tried to make) and 'Semaphores' sub-chapters of **Management REST interface** ... Would like to add **Statistics** as sub-chapter also, all REST that is not /ngsi9, /ngsi10, /v1 nor /v2 ...